test/integration/search.test.ts

```diff
@@ -166,6 +166,21 @@ describe('search', () => {
       }
     }
   });
+
+  test.skip('search all with filters', async () => {
+    const results = await client.search.all('fruits', {
+      filter: { users: { full_name: 'Team fruits' } }
+    });
+
+    for (const result of results) {
+      expect(result.table).toBe('users');
```

Comment:
Need to review this after the backend team finishes their discussion

https://github.com/xataio/xata/pull/825#issuecomment-1162199983